app/.storybook/main.js

```diff
@@ -1,9 +1,14 @@
 const nextConfig = require('../next.config')
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+const path = require('path')
 
 module.exports = {
   stories: ['../stories/**/*.stories.@(mdx|js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-essentials', 'storybook-react-i18next'],
+  addons: [
+    '@storybook/addon-essentials',
+    'storybook-addon-next-router',
```

Comment:
Notes for reviewers: This adds [storybook-addon-next-router](https://storybook.js.org/addons/storybook-addon-next-router) to handle `next/router` in storybook.